spec/spec_helper.rb

```diff
@@ -8,6 +8,18 @@
 # Load Support Files
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
 
+module JavaScriptHelper
```

Comment:
So I found out Rails 3.2, 4.0, 4.1, 4.2, some will translate the `<` to `\\u003E` or `\\u003e`, this helper change these back to `<`. So that in our tests we can assert with one example instead to assert against different version behaviours.